var/spack/repos/builtin/packages/xdotool/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Xdotool(MakefilePackage):
+    """fake keyboard/mouse input, window management, and more"""
+
+    homepage = "https://github.com/jordansissel/xdotool"
+    url      = "https://github.com/jordansissel/xdotool/releases/download/v3.20160805.1/xdotool-3.20160805.1.tar.gz"
+
+    maintainers = ['jordansissel']
```

Comment:
@jordansissel are you okay with being listed as an official maintainer of this package? You don't have to be a Spack expert, it just gives us someone to ping in case a user reports build issues or wants to modify the package.